lib/main.coffee

```diff
@@ -2,6 +2,17 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+cmpVersions = (v1, v2) ->
```

Comment:
Run coala locally and apply the patch on this section to fix the gitmate-bot error